client/src/main/java/gui/instance/ClientAppRunner.java

```diff
@@ -0,0 +1,65 @@
+package gui.instance;
+
+import connection.ServerConnection;
+import gui.data.Player;
+import gui.data.ServerInfo;
+import gui.printers.FleetView;
+
+/**
+ * This is the singleton class. It's purpose is to
+ * provide information flow between controller instances.
+ */
+public class ClientAppRunner {
+
+    private FleetView fleet;
+    private ServerConnection serverConnection;
+    private Player currentPlayer;
+
+    private static ClientAppRunner ourInstance = new ClientAppRunner();
```

Comment:
why singleton?

Reply:
To save some informations through the game scenes.